stable/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 10.5.6
+version: 10.5.7
```

Comment:
This should be at least a minor version as you are adding a new feature. If this PR is going to break the backward compatibility it should be a major change.

Reply:
This doesn't look like a new feature to me. It fixes some annoying bugs by using functionality that already exists in this chart.

Reply:
I stand corrected. Looks like resetting sentinels was never a "feature" of this chart. 
The only difference I observed while using this patched version is that there are no longer any failed pods. Without the changes from this PR we had a randomly failing pod every 2-3 days. The upgrade process worked fine, too.

Reply:
That's fully my opinion - I don't want to introduce any features (the helm chart already promises to provide a HA redis cluster, otherwise it wouldn't need the sentinels), but want to use the chart without the mentioned  problems, I suppose a cluster actally should keep away from me as a cluster consumer.